### DISCUSSION TITLE
Perbedaan Z Score dan Z test

### DISCUSSION THREAD

**Discussion by DhiaFS** [teamalgoritma/community#66] [Q&A] [answered]
Mau tanya, kalau dari studi kasus kita kan z-score untuk hitung probabilitas dalam rumusnya cukup pakai pembagi standar deviasi. Tapi z-test untuk hitung signifikansi kan rumusnya pakai pembagi SE.(standar eror kah?). Nah bisa kah kita menguji p value dari z score juga? walau z score untuk probabilitas bukan signifikansi

**Comment by inytss** [comment, marked as answer]
Dear @DhiaFS ,
Berdasarkan artikel berikut https://tungmphung.com/z-score-z-statistic-z-test-z-distribution/ disebutkan bahwa perbedaan antar Z-score dan Z-test sebagai berikut.

Z-score merupakan nilai suatu x yang kita dianggap berdistribusi normal baku dengan membagikan sebuah nilai rata-rata dengan standar deviasinya sehingga nilainya mendekati normal baku dengan mean = 0 dan standar deviasi 1.
Sedangkan Z-test merupakan sebuah uji hipotesis dengan pendekatan distribusi normal untuk menguji apakah kita dapat menolak mean dengan nilai tertentu berdasarkan suatu nilai standar error tertentu.

Jika dilihat berdasarkan pengertiannya, kedua nilai tersebut memiliki concern yang berbeda, Z-test untuk mentransformasikan data biasa sua=paya lebih mendekati distribusi normal, Z-score untuk melakukan uji hipotesis, apakah sample sudah mewakili populasi.

Sehingga dalam menghitung p-value pada uji hipotesis akan lebih baik jika tetap menggunakan Z-test dengan pembagi SE karena SE disini bertujuan untuk mengetahui apakah sampel data yang digunakan sudah merepresentasikan populasi.

Demikian jawaban dari saya, semoga membantu.
